client/src/components/PitchDoc/ClaimPitchModal.tsx

```diff
@@ -0,0 +1,295 @@
+      const res = await loadUser(pitch.approvedBy);
+      if (!isError(res)) {
+        setApprovedBy(
+          `${res.data.result.firstName} ${res.data.result.lastName}`,
+        );
+      }
+    }
+  }, [
+    pitch.assignmentContributors,
+    pitch.pitchAuthor,
+    pitch.approvedBy,
+    temp_pfp,
+  ]);
+
+  useEffect(() => {
+    getTeams();
+    setSelectedTeams([]);
+    getUser();
+    setPitchLink('');
+  }, [firstOpen, getTeams, getUser]);
+
+  return (
+    <>
+      <Modal
+        className="claim-pitch-modal"
+        onClose={() => {
+          setFirstOpen(false);
+        }}
+        onOpen={() => {
+          setFirstOpen(true);
+        }}
+        open={firstOpen}
+        trigger={<PitchCard pitch={pitch} openTeams={openTeams}></PitchCard>}
+        closeIcon
+      >
+        <Modal.Actions>
+          <div className="claim-pitches-wrapper">
+            <div className="pitch-info">
+              <div className="topics-wrapper">
+                <Label.Group style={{ marginBottom: 10 }} circular>
+                  {pitch.topics.map((topic, idx) => (
+                    <>
+                      <Label
+                        style={{
+                          backgroundColor:
+                            interestsButtons[enumToInterestButtons[topic]],
+                        }}
+                        key={idx}
+                      >
+                        {enumToInterestButtons[topic]}
+                      </Label>
+                    </>
+                  ))}
+                </Label.Group>
+              </div>
+              <div className="pitch-name"> {pitch.name} </div>
+              <span className="submitted-reviewed-by">
+                {' '}
+                Submitted By: {pitchAuthor}
+              </span>
+              <span className="submitted-reviewed-by">
+                {' '}
+                Reviewed By: {approvedBy}
+              </span>
+              <div className="section-description">
+                Here lies a two sentence summary of pitch. It will be two
+                sentences and no more.
+              </div>
+            </div>
+            <div className="pitch-link">
+              <div className="pitch-link-title">Link to Pitch:</div>
+              <Input
+                className="pitch-link-input"
+                placeholder="Pitch link"
+                onChange={(e) => {
+                  setPitchLink(e.currentTarget.value);
+                }}
+              />
+            </div>
+            <div className="role-contributor-section">
+              <div className="section-title">Roles Available Per Team:</div>
+              <div className="section-description">
+                Please select the team that you will be filling the role for
+                before claiming this pitch.
+              </div>
+              <div className="role-items">
+                {Object.entries(openTeams).map(([team, value], idx) => (
+                  <>
```

Comment:
I don't think you need these empty JSX fragments if you are already wrapping the entire thing in a div